test/box-luatest/gh_7362_net_box_async_ping_test.lua

```diff
@@ -0,0 +1,40 @@
+local net = require('net.box')
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+local g = t.group()
+
+local TIMEOUT = 5
+
+g.before_all(function()
+    g.server = server:new({alias = 'server'})
+    g.server:start()
+    g.conn = net.connect(g.server.net_box_uri)
+end)
+
+g.before_test('test_disconnected_ping', function(g)
+    g.server:stop()
+end)
+
+g.after_all(function()
+    g.conn:close()
+end)
+
+g.test_connected_ping = function()
```

Comment:
If this test case would be run after `test_disconnected_ping`, then you would have the server stopped and the pings below will fail. I suggest you either merge these test cases into one function to control the order, or add server `:start()` after `test_disconnected_ping`.